phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/setting/skin/AboutPanelSkin.java

```diff
@@ -0,0 +1,147 @@
+
+        final Label nameLabel = new Label();
+        nameLabel.textProperty().bind(
+                Bindings.createStringBinding(() -> Optional.ofNullable(getControl().getBuildInformation())
+                        .map(ApplicationBuildInformation::getApplicationName).orElse(null),
+                        getControl().buildInformationProperty()));
+
+        aboutGrid.addRow(0, nameDescription, nameLabel);
+
+        final Text versionDescription = new Text(tr("Version:"));
+        versionDescription.getStyleClass().add("captionTitle");
+
+        final Label versionLabel = new Label();
+        versionLabel.textProperty().bind(
+                Bindings.createStringBinding(() -> Optional.ofNullable(getControl().getBuildInformation())
+                        .map(ApplicationBuildInformation::getApplicationVersion).orElse(null),
+                        getControl().buildInformationProperty()));
+
+        aboutGrid.addRow(1, versionDescription, versionLabel);
+
+        final Text gitRevisionDescription = new Text(tr("Git Revision:"));
+        gitRevisionDescription.getStyleClass().add("captionTitle");
+
+        final Hyperlink gitRevisionHyperlink = createGitRevisionHyperlink();
+
+        aboutGrid.addRow(2, gitRevisionDescription, gitRevisionHyperlink);
+
+        final Text buildTimestampDescription = new Text(tr("Build Timestamp:"));
+        buildTimestampDescription.getStyleClass().add("captionTitle");
+
+        final Label buildTimestampLabel = new Label();
+        buildTimestampLabel.textProperty().bind(
+                Bindings.createStringBinding(() -> Optional.ofNullable(getControl().getBuildInformation())
+                        .map(ApplicationBuildInformation::getApplicationBuildTimestamp).orElse(null),
+                        getControl().buildInformationProperty()));
+
+        aboutGrid.addRow(3, buildTimestampDescription, buildTimestampLabel);
+
+        return aboutGrid;
+    }
+
+    /**
+     * Creates the {@link Hyperlink} leading to the git revision on github
+     *
+     * @return The {@link Hyperlink} leading to the git revision on github
+     */
+    private Hyperlink createGitRevisionHyperlink() {
+        final StringBinding revisionText = Bindings.createStringBinding(
+                () -> Optional.ofNullable(getControl().getBuildInformation())
+                        .map(ApplicationBuildInformation::getApplicationGitRevision).orElse(null),
+                getControl().buildInformationProperty());
+
+        final BooleanBinding disableProperty = Bindings.when(Bindings
+                .and(Bindings.isNotNull(revisionText), Bindings.notEqual(revisionText, "unknown")))
+                .then(false).otherwise(true);
+
+        final Hyperlink gitRevisionHyperlink = new Hyperlink();
+
+        gitRevisionHyperlink.textProperty().bind(revisionText);
+
+        // if the git revision equals "unknown" disable the hyperlink
+        gitRevisionHyperlink.disableProperty().bind(disableProperty);
```

Comment:
Does it make sense to handle this with bindings? The information presented on the about panel is static.

Reply:
I see no disadvantages of using bindings in this case. In addition we don't know whether we will need to change the shown version sometime in the future at runtime.
Maybe we'll add an integrated updater later on

Reply:
Ok.